src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,192 @@
+      let registration = await getRegistration();
+      // Await the outcome of the unregistration attempt
+      // so that the UI update is not superceded by a 
+      // returning Promise.
+      await registration.unregister();
+      updateUI();
+    }
+    ```
+
+In the tab where you're viewing the live app, reload the page. The **Register service worker** and **Unregister service worker** buttons should now be working.
+
+## Send notifications to the service worker
+
+In this step, you'll write code that will run when the user clicks **Send a notification** in the app UI. This code will create a notification, check that a service worker is registered, and then send the notification to the service worker using its `postMessage` method.
+
+In the embedded Glitch editor, open `public/index.js` and
+replace the `sendNotification` function with the following code:
+
+```js
+// Create and send a test notification to the service worker.
+async function sendNotification() {
+  // Use a random number as part of the notification data
+  // (so you can tell the notifications apart during testing!)
+  let randy = Math.floor(Math.random() * 100);
+  let notification = {
+    title: 'Test ' + randy,
+    options: { body: 'Test body ' + randy }
+  };
+  // Get a reference to the service worker registration.
+  let registration = await getRegistration();
+  // Check that the service worker registration exists.
+  if (registration) {
+    // Check that a service worker controller exists before
+    // trying to access the postMessage method.
+    if (navigator.serviceWorker.controller) {
+      navigator.serviceWorker.controller.postMessage(notification);
+    } else {
+      console.log('No service worker controller found. Try a soft reload.');
+    }
+  }
+}
+```
+
+Here's what that code is doing:
+
+*   `sendNotification` is an asynchronous function, so you can use `await` to get a reference to the service worker registration.
+
+*   The service worker's `postMessage` method sends data from the app to the service worker. See the [MDN documentation on  postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage) for more information.
+
+*   The code checks for the presence of the `navigator.serviceWorker.controller` property before trying to access the `postMessage` function. `navigator.serviceWorker.controller` will be `null` if there is no active service worker, or if the page has been force refreshed (`Shift+`**Reload**). See the [ServiceWorker controller documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/controller) for more information.
+
+## Handle notifications in the service worker
+
+In this step, you'll write code in the service worker that will handle messages posted to it and display notifications to the user.
+
+In the embedded Glitch editor, open `public/service-worker.js`. Add the following code to the end of the file:
```

Comment:
Since this is a one-item list, I just made it a paragraph, which leaves more horizontal space for this and the following content.